core/internal/cltest/factories.go

```diff
@@ -727,7 +736,7 @@ func MustGenerateRandomKey(t testing.TB, opts ...interface{}) ethkey.Key {
 
 	key := ethkey.Key{
 		Address:   eip,
-		JSON:      gormpostgrestypes.Jsonb{RawMessage: keyjsonbytes},
+		JSON:      datatypes.JSON(keyjsonbytes),
```

Comment:
oh nice - I think earlier I was under the impression the gorm v2 datatypes didn't support jsonb, but looks like they default to that for postgres https://github.com/go-gorm/datatypes/blob/master/json.go#L75